desktop-widgets/command.cpp

```diff
@@ -2,10 +2,12 @@
 
 #include "command.h"
 #include "command_divelist.h"
+#include "command_edit.h"
 
 namespace Command {
 
 // Dive-list related commands
+
```

Comment:
why an empty?

Also, there's a typo in the commit message, and as we all know by now you can't comment on those, so I'll comment here that the commit message of "Undo: implement rudimentary undo of dive-notes editing" using "editig" instead of "editing"...

Reply:
Removed.